netutils/config/clean.py

```diff
@@ -1,11 +1,11 @@
 """Functions for working with configuration to clean the config."""
 # pylint: disable=anomalous-backslash-in-string
 
-
 import re
+import typing as t
 
 
-def clean_config(config, filters):
+def clean_config(config: str, filters: t.List[str]) -> str:
```

Comment:
The docstring example for this doesn't pass type checking. Now AFAIK there is no way for Mypy to perform type checking on the code in the examples, but I still think it's worth pointing out: `clean_filters` in that example is a `List[Dict[str, str]]` rather than the `List[str]` annotation in the function signature here.